docs/tracing/10-01-traces-troubleshooting.md

```diff
@@ -5,14 +5,41 @@ type: Troubleshooting
 
 ## Jaeger shows only a few traces
 
-Istio Pilot sets the trace sampling rate at `1`, where `100` is the maximum value. This means that only 1 out of 100 requests is sent to Jaeger for trace recording. To change this system behavior, run:
+By default, the **PILOT_TRACE_SAMPLING** value in the [IstioControlPlane](https://istio.io/docs/reference/config/istio.operator.v1alpha12.pb/) is set to `1`, where `100` is the maximum value. This means that only 1 out of 100 requests is sent to Jaeger for trace recording.
+To change this system behavior, [override](/root/kyma/#configuration-helm-overrides-for-kyma-installation) the existing configuration with a custom control plane definition.
+
+1. Add and apply a ConfigMap in the `kyma-installer` Namespace in which you set the value for the **PILOT_TRACE_SAMPLING** attribute to `60`.
```

Comment:
Thinking out loud, shouldn't this be also part of production profile for monitoring?

Reply:
I will talk to @a-thaler  about it :)